app/pages/project/classify.jsx

```diff
@@ -182,7 +184,7 @@ export class ProjectClassifyPage extends React.Component {
   loadAnotherSubject() {
     const { actions, project, workflow } = this.props;
     if (workflow) {
-      actions.classifier.nextSubject(project, workflow);
+      actions.classifier.nextSubject(project);
```

Comment:
Out of interest, why was the workflow removed here?

Reply:
Per https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/redux/ducks/classify.js#L315-L320 I don't think the workflow param is used? The classify duck tracks workflow in it's state (https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/redux/ducks/classify.js#L164)?

Reply:
This also applies to line 123.